packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -387,8 +385,10 @@ export class AppStudioPluginImpl {
       if (await this.checkFileExist(archiveOutlineFile)) {
         await fs.copyFile(archiveOutlineFile, newOutlineFile);
       }
+      return { enableAuth: !!manifest.webApplicationInfo };
```

Comment:
For SSO, do we care empty `webApplicationInfo` or `webApplicationInfo.id` is required?

Reply:
Yes, webApplicationInfo.id is more accurate. 
Changed it to !!manifest?.webApplicationInfo?.id